package.json

```diff
@@ -6,8 +6,9 @@
   "description": "A safe home for all your data",
   "main": "core/core.bundle.js",
   "scripts": {
-    "watch": "webpack --progress --colors --watch --mode development --config webpack.dev.js",
-    "build": "webpack -p --config webpack.prod.js",
+    "watch": "npm run build:dev && webpack-dev-server --progress --colors --watch --config webpack.dev.js --public $SERVER_HOST",
```

Comment:
whats the format of SERVER_HOST? @tempelgogo @scramb

Reply:
domainname/ip:port